src/status_im2/contexts/chat/lightbox/zoomable_image/constants.cljs

```diff
@@ -15,3 +15,5 @@
 (def ^:const default-duration 300)
 
 (def ^:const default-dimension 1000)
+
+(def ^:const margin 8)
```

Comment:
Shouldn't this be in the `style` namespace?

Reply:
The reason I put as in a constant file is because this is not actually used as a `margin` style. It is used in other calculations to calculate width and height, and to scale on tap